tests/cases/standalone/common/alter/change_col_inverted_index.sql

```diff
@@ -0,0 +1,36 @@
+CREATE TABLE fox (
+    ts TIMESTAMP TIME INDEX,
+    fox STRING,
+);
```

Comment:
You can add some case:

```
CREATE TABLE test_pk (..., foo STRING, bar INT, PRIMARY KEY (foo, bar));
SHOW INDEX;
UNSET foo;
SHOW INDEX;
UNSET bar;
SHOW INDEX;
SET foo;
SHOW INDEX;
```

Reply:
added the case the results looks a bit strange to me. If its not what we expected i can debug more.

Reply:
modified the logic to produce expected results